hazelcast-client/src/test/java/com/hazelcast/client/test/TestHazelcastFactory.java

```diff
@@ -47,7 +47,11 @@
     private final TestClientRegistry clientRegistry;
 
     public TestHazelcastFactory() {
-        super(0);
+        this(0);
+    }
+
+    public TestHazelcastFactory(int count) {
```

Comment:
There is no need to use count in TestHazelcastFactory. 
I deliberately did not add such a constructor to avoid confusion. 
Do you really see a need here ?

Reply:
To me it looked like an omission. The superclass has it. Without it, `newInstances()` method is useless. What confusion?

Reply:
Ah, there are already methods that uses this count. In that case, it make sense to have it. It is just, we have more than one way of doing same thing, and it is confusing to me. 
for example, one can call this one to achieve same thing. 
`newInstances(Config config, int nodeCount) {`
May be TestHazelcastFactory should not have extended TestHazelcastInstanceFactory in the first place, not sure. ?
In short, I am OK with the change.